### PR TITLE
Add Elsha to Before I Die

### PR DESCRIPTION
**Pull request by elshaek**
Adding Elsha to Before I Die with text and images.

### CODE REVIEW

File: CONTRIBUTORS.md
Comment (XanderRubio) [inline CONTRIBUTORS.md:1]: Hi @elshaek. Confirming you did everything correctly, I just needed to update the list to avoid a merge conflict.

**Review by XanderRubio — APPROVED** [review]
Everything looks great on the preview link and thank you for sharing your images and what you want to do before you die @elshaek! I am going ahead with merging now.

File: CONTRIBUTORS.md
Comment (elshaek) [reply CONTRIBUTORS.md:1]: Sorry, I didn't see your message earlier and tried to resolve the conflicts... Not sure if that messed things up

Comment (XanderRubio) [reply CONTRIBUTORS.md:1]: All good!😀

**Comment by XanderRubio** [comment]
@elshaek   Your pull request has been successfully merged!!! Congratulations Bottle with Popping CorkBottle with Popping CorkBottle with Popping CorkDouble Exclamation MarkDouble Exclamation MarkDouble Exclamation Mark.

 If you haven't already, please go ahead and star Glowing StarGlowing StarGlowing Starthis repo and you are more than welcome to continually contribute to this project by assisting with our [open issues](https://github.com/BeforeIDieCode/BeforeIDieAchievements/issues), or adding your own new open issues, and additionally you can get an idea of ideas we have to continually grow this project further as technology and tech stack continually evolve by looking at the [ROADMAP](https://github.com/BeforeIDieCode/BeforeIDieAchievements/blob/main/ROADMAP.md) for the project. Have a great day, and thank you for taking the time to contribute, 

Elsha! You can view your contribution now on the [live link](https://before-i-die-achievements.vercel.app/). Have a great rest of your day, and I will be sending you an invite to be a member of the Before I Die Code now.

**Pull request merged**